### DISCUSSION TITLE
Time stamp

### DISCUSSION THREAD

**Discussion by picotrain77** [blakeblackshear/frigate#1575] [Ask A Question] [answered]
Hi and firstly I would like to thank you for by far the best integration I have ever used, I take my hat off to you sir.

Apologies if this has been asked or mentioned already but a quick search did not show up anything.

The time stamp on my captures is white and for me due to lots of bright white in the camera shots is almost unreadable - so it there a way to change this to black and or change font size etc.

thanks again and keep up the great work

**Comment by scooper1** [comment, marked as answer]
The new beta has a time stamp section where you can change colour

There maybe a bug or doc error as got errors using RGB  values of less then 255.       You can have nothing but white 😃
